src/Service.Host/client/src/configureStore.js

```diff
@@ -5,7 +5,8 @@ import reducer from './reducers';
 import authorization from './middleware/authorization';
 import itemCreated from './middleware/itemCreated';
 
-const composeEnhancers = window['__REDUX_DEVTOOLS_EXTENSION_COMPOSE__'] || compose;
+// eslint-disable-next-line no-underscore-dangle
```

Comment:
remove?

Reply:
I've used dot notation to get rid of another eslinter error and then i got the one above.. it's one or the other